libtuf-server/src/main/scala/com/advancedtelematic/libtuf_server/data/Entities.scala

```diff
@@ -0,0 +1,16 @@
+package com.advancedtelematic.libtuf_server.data
```

Comment:
I think this could just be in the resource file? We don't use it anywhere else, maybe it tests, but there it can be imported from the resource file.